npm-debug.log

```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```

Comment:
If you add this to .gitignore and you can remove it from source and it won't come back.